ocpmodels/trainers/base_trainer.py

```diff
@@ -55,12 +54,26 @@ def __init__(
         name="base_trainer",
     ):
         self.name = name
+        if torch.cuda.is_available():
+            self.device = local_rank
+        else:
+            self.device = "cpu"
 
         if run_dir is None:
             run_dir = os.getcwd()
         run_dir = Path(run_dir)
 
-        timestamp = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+        _timestamp = torch.tensor(
+            int(datetime.datetime.now().strftime("%Y%m%d%H%M%S"))
+        ).cuda(self.device)
```

Comment:
Should this be `.to(self.device)`?

Reply:
Either work, but yeah `.to()` is new/cleaner. Fixed.